packages/editor-ui/src/stores/pushConnection.store.ts

```diff
@@ -33,10 +33,11 @@ export const usePushConnectionStore = defineStore(STORES.PUSH, () => {
 	const lostConnection = ref(false);
 	const outgoingQueue = ref<unknown[]>([]);
 	const isConnectionOpen = ref(false);
-	const onMessageReceivedHandlers = ref<OnPushMessageHandler[]>([]);
+
+	const onMessageReceivedHandlers = ref<OnPushMessageHandler>();
```

Comment:
Unfortunately we can't change this to be only a single handler, as also `CollaborationStore` is depending on these messages. This would cause a race condition between `CollaborationStore` and `PushConnectionMixin` on who gets to receive the messages